pages/getting_started/planning.rst

```diff
@@ -75,7 +75,11 @@ Collection method
 ^^^^^^^^^^^^^^^^^
 **"How will you collect it?"**
  
-After a list of event sources has been determined, the next step is to decide the method of collection for each source.  Although many hardware and software products support common methods such as sending log data via syslog, many do not. It is critical to understand what method each event source uses and what resources that may require. For example, if a log shipper will be required to read logs from a local file on all servers, a log shipper must be selected and tested prior to deployment. In other cases, proprietary API’s or software tools must be employed and integrated.
+After a list of event sources has been determined, the next step is to decide the method of collection for each source.  Although many hardware and software products support common methods such as sending log data via syslog, many do not. 
+Understanding the answer to these questions is critical: 
+*What method does each event source use?
+*What resources are required? 
```

Comment:
```suggestion
* What method does each event source use?
* What resources are required? 
```

Reply:
@rhoughton thanks for contributing! Just a note that for bulleted lists to work, they'll need a space between the * and the rest of the characters.

Reply:
oic, thank you! .rst files are cool, I'm learning :)